tests/test-admin-deploy-karg.sh

```diff
@@ -77,3 +77,10 @@ assert_not_file_has_content sysroot/boot/loader/entries/ostree-2-testos.conf 'op
 assert_not_file_has_content sysroot/boot/loader/entries/ostree-2-testos.conf 'options.*APPENDARG=VALAPPEND'
 
 echo "ok deploy --karg-delete"
+
+${CMD_PREFIX} ostree admin deploy  --os=testos --karg-append="test=\"1 2\"" testos:testos/buildmain/x86_64-runtime
+assert_file_has_content sysroot/boot/loader/entries/ostree-2-testos.conf 'options.*test="1 2"'
+${CMD_PREFIX} ostree admin deploy  --os=testos --karg-delete="test=\"1 2\"" testos:testos/buildmain/x86_64-runtime
```

Comment:
The delete interface might be a little simpler/cleaner if it expected key only (the test part)